packages/IntegrationTest/windows/integrationtest/TestHostHarness.h

```diff
@@ -41,6 +41,11 @@ class TestHostHarness : public winrt::implements<TestHostHarness, winrt::Windows
   winrt::com_ptr<TestTransaction> m_currentTransaction;
   winrt::Microsoft::ReactNative::IRedBoxHandler m_redboxHandler;
   std::optional<TestCommandResponse> m_pendingResponse;
+
+  bool m_instanceFailedToLoad{false};
```

Comment:
Would this be better 'inverted' i.e. considering things failed until proven load succeeded? 
Or are these cases whithout load.?

Reply:
The instance is loaded asynchronously after harness creation, so the state at construction would be that we haven't attempted to load yet.